.github/workflows/artifacts.yml

```diff
@@ -14,7 +14,7 @@ jobs:
     runs-on: ubuntu-18.04
     container:
       # Older Ubuntu for older glibc
```

Comment:
this comment is meaningful. artifcts build on bionic don't work on xenial.

Yet, IIRC artifacts are built by gitlab CI, aren't they; then the whole workflow could be removed. (as well as release.py which isn't used by that scripts).

Reply:
Yes, I understand the comment, but Xenial is just too old, not even the ppa supports it, it seems, hence the errors. So Focal is now the new "older glibc" container and it seems to work.

However, it's even better to remove things, let so let me try just that.